test-lit/unit/lit.local.cfg

```diff
@@ -0,0 +1 @@
+config.suffixes = ['.f90', '.F90']
```

Comment:
There are some tests in the test/ directory that use '.f' and '.F' as well, do those need to be included as suffixes here?

Reply:
Probably yes, I only add the bare minimum. I believe as the tests are being added to the test-lit this  file should reflect this change as well as the others.

Reply:
Why not add them now and save some poor soul in the future from having to figure this out again?

Reply:
documentation/OptionComparison.md has a list of file suffixes that other compilers accept, e.g. Cray accepts .f03 and .ftn.  There's also a list in tools/f18.cc: "f", "F", "f18", "F18", "f90", "F90", "f95", "F95", "ff", "ff18", "ff90", and "ff95".  The NAG compiler accept the 'ff' suffixes too.

Reply:
Ok!

Reply:
Done